packages/client/README.md

```diff
@@ -0,0 +1,49 @@
+# @ucanto/client
+
+`@ucanto/client` provides the tools necessary to create, sign, and send UCAN-based RPC invocations. It enables secure communication with UCAN-compliant services while ensuring proper authorization and delegation handling.
+
+## What It Provides
+- **UCAN Invocation Handling**: Creates and signs capability invocations.
+- **Batch Invocation Support**: Enables multiple invocations in a single request.
+- **Secure Communication**: Ensures interactions are cryptographically signed and verified.
+
+## How It Fits with Other Modules
+- [`@ucanto/core`](../core/README.md): Defines capability structures and execution logic.
+- [`@ucanto/server`](../server/README.md): Processes invocations received from the client.
+- [`@ucanto/interface`](../interface/README.md): Provides shared types for request and response handling.
+- [`@ucanto/principal`](../principal/README.md): Manages cryptographic signing for invocations.
+- [`@ucanto/transport`](../transport/README.md): Handles encoding and sending of requests.
```

Comment:
I'm not sure how these relative links work when published on npm. I assume it handles them but we might have to tweak later.